08-kubernetes-sealed-secrets/README.md

```diff
@@ -103,19 +103,19 @@ The output looks similar to:
 
 ```text
 NAME                            CHART VERSION   APP VERSION     DESCRIPTION
-sealed-secrets/sealed-secrets   2.1.6          v0.17.5         Helm chart for the sealed-secrets controller.
+sealed-secrets/sealed-secrets   2.4.0           v0.18.1         Helm chart for the sealed-secrets controller.
```

Comment:
After running:

```console
helm repo add sealed-secrets https://bitnami-labs.github.io/sealed-secrets
helm repo update sealed-secrets
helm search repo sealed-secrets
```

I get:

```text
NAME                         	CHART VERSION	APP VERSION	DESCRIPTION                                       
bakito/sealed-secrets-web    	2.8.0        	v2.8.0     	A web interface for Sealed Secrets by Bitnami.    
bitnami/sealed-secrets       	1.0.10       	0.18.1     	Sealed Secrets are "one-way" encrypted K8s Secr...
sealed-secrets/sealed-secrets	2.5.2        	v0.18.1    	Helm chart for the sealed-secrets controller.
```

I see that **v2.5.2** of the **sealed-secrets/sealed-secrets** Helm chart is mapped to the **v0.18.1** of the app. Can you double check please?

Reply:
Double checked. After I upgrade to 2.4.0 and do an `helm ls -n sealed-secrets` I get this:
<img width="1110" alt="Screenshot 2022-08-03 at 14 15 45" src="https://user-images.githubusercontent.com/104416379/182595166-ea05b5b4-4a05-4fb6-bb32-f9684316205c.png">
Apparently the app version has not changed in a while.